File: backend/src/main/java/com/tjh/riskfactor/entity/view/TaskView.java
package com.tjh.riskfactor.entity.view;

import com.fasterxml.jackson.annotation.JsonFormat;

import java.util.Date;

public interface TaskView {

    Integer getId();

    String getName();

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
    Date getMtime();

    String getCenter();
}

File: backend/src/main/kotlin/com/tjh/riskfactor/entity/view/TaskView.kt
package com.tjh.riskfactor.entity.view

import com.fasterxml.jackson.annotation.JsonFormat

import java.util.Date

interface TaskView {
    val id: Int
    val name: String
    val center: String
    @get:JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
    val mtime: Date
}
